src/test/java/org/skife/jdbi/v2/sqlobject/TestGetGeneratedKeysPostgres.java

```diff
@@ -67,9 +72,10 @@ public Object withHandle(Handle handle) throws Exception
         public String findNameById(@Bind long id);
     }
 
-    @Ignore
     @Test
     public void testFoo() throws Exception {
+        assumeTrue(postgresIsInstalled);
```

Comment:
Would this be better in the `@Before` ?  Gets rid of the if blocks entirely, and should work just the same: https://github.com/junit-team/junit/wiki/Assumptions-with-assume